pom.xml

```diff
@@ -68,7 +68,7 @@
       <version>${folio-spring-base.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.folio</groupId>
+      <groupId>com.github.folio-org</groupId>
```

Comment:
Should this be changed? In the lib-fqm-query-processor PR, it looks like the group is still set to `org.folio`

Reply:
see the above comment; this uses jitpack.io to temporarily pull from github instead of the normal FOLIO repository (since that reop only contains changes in master). Added this to test in CI; I'll revert it once the lib-fqm-query-processor PR is merged